app/style/Tutorial.less

```diff
@@ -117,6 +64,22 @@
       margin: 0em 5em 2em 0em;
       break-inside: avoid-column;
     }
+
+    ul {
+      margin: 0em 5em 2em -3.1em;
```

Comment:
margin-right should be 2em

for `<p>` too

Reply:
That seems to space out the text better so it won't push down the Staking and Governance buttons